README.md

```diff
@@ -1,9 +1,21 @@
-[![Version Badge](https://anaconda.org/conda-forge/spglib/badges/version.svg)](https://anaconda.org/conda-forge/spglib)
-[![Downloads Badge](https://anaconda.org/conda-forge/spglib/badges/downloads.svg)](https://anaconda.org/conda-forge/spglib)
-[![PyPI](https://img.shields.io/pypi/dm/spglib.svg?maxAge=2592000)](https://pypi.python.org/pypi/spglib)
-
 # Spglib
 
+[![CI Status][ci-badge]][ci-link]
+[![Documentation Status][rtd-badge]][rtd-link]
+[![Codecov Status][codecov-badge]][codecov-link]
+
+![License Status][license-badge]
+![CMake Status][cmake-badge]
+[![Python Versions][python-version]][pypi-link]
+
+[![Release Status][release-badge]][release-link]
+[![Conda Status][conda-badge]][conda-link]
+[![PyPI Status][pypi-badge]][pypi-link]
+[![Fedora Status][fedora-badge]][fedora-link]
+
+[![PyPI Downloads][pypi-downloads]][pypi-link]
+[![Conda Downloads][conda-downloads]][conda-link]
```

Comment:
Most badges are from https://shields.io/badges. There could be more interesting ones to find. I think a social one would be good, but there is no matrix/discord room etc. Doesn't have to be spglib specific, just a place where developers can be contacted in real-time

Reply:
I think the GitHub issue and discussion work as such a community room now. So, I do not feel the need for another social one so much.